expandablerecyclerview/src/main/java/com/bignerdranch/expandablerecyclerview/Adapter/ExpandableRecyclerAdapter.java

```diff
@@ -144,57 +196,57 @@ public void onBindViewHolder(RecyclerView.ViewHolder holder, int position) {
 
     /**
      * Callback called from {@link #onCreateViewHolder(ViewGroup, int)} when
      * the list item created is a parent.
      *
-     * @param parentViewGroup The {@link ViewGroup} in the list for which a {@link PVH}
+     * @param parentViewGroup The {@link ViewGroup} in the list for which a {@link ParentViewHolder}
      *                        is being created
      * @return A {@code PVH} corresponding to the {@link ParentListItem} with
-     *         the {@code ViewGroup} parentViewGroup
+     * the {@code ViewGroup} parentViewGroup
      */
-    public abstract PVH onCreateParentViewHolder(ViewGroup parentViewGroup);
+    public abstract ParentViewHolder onCreateParentViewHolder(ViewGroup parentViewGroup, int viewType);
 
     /**
      * Callback called from {@link #onCreateViewHolder(ViewGroup, int)} when
      * the list item created is a child.
      *
-     * @param childViewGroup The {@link ViewGroup} in the list for which a {@link CVH}
+     * @param childViewGroup The {@link ViewGroup} in the list for which a {@link ChildViewHolder}
      *                       is being created
      * @return A {@code CVH} corresponding to the child list item with the
-     *         {@code ViewGroup} childViewGroup
+     * {@code ViewGroup} childViewGroup
      */
-    public abstract CVH onCreateChildViewHolder(ViewGroup childViewGroup);
+    public abstract ChildViewHolder onCreateChildViewHolder(ViewGroup childViewGroup, int viewType);
 
     /**
      * Callback called from onBindViewHolder(RecyclerView.ViewHolder, int)
      * when the list item bound to is a parent.
      * <p>
-     * Bind data to the {@link PVH} here.
+     * Bind data to the {@link ParentViewHolder} here.
      *
      * @param parentViewHolder The {@code PVH} to bind data to
      * @param position The index in the list at which to bind
      * @param parentListItem The {@link ParentListItem} which holds the data to
      *                       be bound to the {@code PVH}
      */
-    public abstract void onBindParentViewHolder(PVH parentViewHolder, int position, ParentListItem parentListItem);
+    public abstract void onBindParentViewHolder(ParentViewHolder parentViewHolder, int position, ParentListItem parentListItem);
 
     /**
      * Callback called from onBindViewHolder(RecyclerView.ViewHolder, int)
      * when the list item bound to is a child.
      * <p>
-     * Bind data to the {@link CVH} here.
+     * Bind data to the {@link ChildViewHolder} here.
      *
      * @param childViewHolder The {@code CVH} to bind data to
      * @param position The index in the list at which to bind
      * @param childListItem The child list item which holds that data to be
      *                      bound to the {@code CVH}
      */
-    public abstract void onBindChildViewHolder(CVH childViewHolder, int position, Object childListItem);
+    public abstract void onBindChildViewHolder(ChildViewHolder childViewHolder, int position, Object childListItem);
```

Comment:
I like that these are more explicit but there's a chance that the repo owner won't like it. If we do want to create a PR for them then it's best not to do these changes in the future for better chances of getting it merged.